### PR TITLE
Timezone banner tooltip

### PR DESCRIPTION
**Pull request by bthaile**
https://github.com/AugurProject/augur/issues/1486

When user logs in the market banner is dismissible.

### CODE REVIEW

File: src/modules/market/components/market-header-message/market-header-message.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            >
+              <div
+                style={{ color: "#372e4b" }}
+                className={Styles.MarketHeaderMessage__tooltip}
+              >
+                {
+                  <div>
+                    <p>
+                      Markets on Augur are created by the community, this means
+                      that errors are can be made in the creation of a market,
```
Comment (phoebemirman) [inline src/modules/market/components/market-header-message/market-header-message.jsx]: "are" and "can" should not both be here. get rid of "are"

File: src/modules/market/components/market-header-message/market-header-message.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                      which can result in the market being resolved as invalid.
+                      If a market resolves as invalid, each share is refunded to
+                      traders in equally amounts.
```
Comment (phoebemirman) [inline src/modules/market/components/market-header-message/market-header-message.jsx]: should be "equal"

File: src/modules/market/components/market-header-message/market-header-message.jsx (hunk 1 above)
Comment (bthaile) [reply src/modules/market/components/market-header-message/market-header-message.jsx]: `  that errors can be made in the creation of a market,` this does read better. good one

File: src/modules/market/components/market-header-message/market-header-message.jsx (hunk 2 above)
Comment (bthaile) [reply src/modules/market/components/market-header-message/market-header-message.jsx]: fixed

**Pull request merged**